#!/bin/sh /usr/share/dpatch/dpatch-run
## debian/quilt/micro_httpd.8-name-change.dpatch by Jari Aalto <jari.aalto@cante.net>
##
## DP: micro_httpd => micro-httpd

@DPATCH@

Index: micro-httpd/micro_httpd.8
===================================================================
--- micro-httpd.orig/micro_httpd.8	2007-01-25 23:25:15.000000000 +0200
+++ micro-httpd/micro_httpd.8	2007-01-25 23:25:30.000000000 +0200
@@ -1,12 +1,12 @@
-.TH micro_httpd 8 "15 March 1999"
+.TH micro-httpd 8 "15 March 1999"
 .SH NAME
-micro_httpd - really small HTTP server
+micro-httpd - really small HTTP server
 .SH SYNOPSIS
-.B micro_httpd
+.B micro-httpd
 .I directory
 .SH DESCRIPTION
 .PP
-.I micro_httpd
+.I micro-httpd
 is a very small HTTP server.
 It runs from inetd, which means its performance is poor.
 But for low-traffic sites, it's quite adequate.
@@ -31,13 +31,13 @@
 .PP
 To install it, add a line like this to /etc/inetd.conf:
 .nf
-    micro_http  stream tcp nowait nobody  /usr/local/sbin/micro_httpd micro_httpd dir
+    micro-http  stream tcp nowait nobody  /usr/local/sbin/micro-httpd micro-httpd dir
 .fi
 Make sure the path to the executable is correct, and change "dir" to be
 the directory you want to serve.
 Then add a line like this to /etc/services:
 .nf
-    micro_http   port/tcp   #Micro HTTP server
+    micro-http   port/tcp   #Micro HTTP server
 .fi
 Change "port" to the port number you want to use - 80, 8000, whatever.
 Then restart inetd by sending it a "HUP" signal, or rebooting.
@@ -53,15 +53,15 @@
 On some Linux systems, you can set the limit on a per-service basis
 in inetd.conf, by changing "nowait" to "nowait.10000".
 .PP
-Note that you can use micro_httpd to serve HTTPS, if you like, by running
+Note that you can use micro-httpd to serve HTTPS, if you like, by running
 it from stunnel.
 First fetch and install stunnel - FreeBSD users can just go to
 /usr/ports/security/stunnel and do a "make cert ; make install".
 Then as root run:
 .nf
     stunnel -p /usr/local/certs/stunnel.pem -d 443 \
-	-l /usr/local/sbin/micro_httpd -- \
-	micro_httpd dir
+	-l /usr/local/sbin/micro-httpd -- \
+	micro-httpd dir
 .fi
 Make sure the paths to the certificate and executable are correct, and
 again don't forget to change "dir" to the directory you want to serve.
